hrm-service/src/config/db.ts

```diff
@@ -1,17 +1,12 @@
-require('dotenv').config();
+import env from 'dotenv';
 
-const config = {
+env.config();
+
+export default {
   username: process.env.RDS_USERNAME || 'hrm',
   password: process.env.RDS_PASSWORD || null,
   database: process.env.RDS_DBNAME || 'hrmdb',
   host: process.env.RDS_HOSTNAME || 'localhost',
   port: process.env.POSTGRES_PORT || 5432,
   dialect: 'postgres',
 };
```

Comment:
Isn't this identical to `hrm-service/db-migrate.js`. Maybe we could rely on one of them only?

Reply:
We did before, but now the support script is JS and this is TS. Also it's nice if the support scripts aren't dipping into application code if they can be kept separate. I thought overall a little duplication was worth it to have the separation